src/upload/index.js

```diff
@@ -2,32 +2,49 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const {
+ const {
     TELEMETRY_ENDPOINT,
     PENDING_PINGS_STORAGE_KEY,
     RECOVERABLE_UPLOAD_ERROR_TIMEOUT
-} = require("./constants");
-const { transformItemWithDefault } = require("./utils");
+} = require("../constants");
+const { transformItemWithDefault } = require("../utils");
+
+// The issue with this approach is that webpack wil bundle all of these even if they are not used.
```

Comment:
Unless we explicitly tell webpack to not do that. Should add this to the comment.